popup.js

```diff
@@ -0,0 +1,138 @@
+mainProject.appendChild(projectData);
+
+const projectParagraph = document.createElement('p');
+projectParagraph.setAttribute('class', 'popup-paragraph');
+projectData.appendChild(projectParagraph);
+projectParagraph.innerText = "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent";
+
+const projectListButton = document.createElement('div');
+projectListButton.setAttribute('class', 'popup-list-button');
+projectData.appendChild(projectListButton);
+
+const projectUl = document.createElement('ul');
+projectUl.setAttribute('class', 'popup-list');
+const languages = ['HTML', 'CSS', 'JavaScript', 'Rugby', 'Bootstrap'];
+
+// Inserting elements in project-data Ul div --> li
+
+for (let i = 0; i < languages.length; i++) {
+  const listItem = document.createElement('li');
+  listItem.setAttribute('class', 'lang-li');
+  listItem.textContent = languages[i];
+  projectUl.appendChild(listItem);
+}
+projectListButton.append(projectUl);
+
+// Inserting elements in project data --> button
+const lastButton = document.createElement('div');
+lastButton.setAttribute('class', 'last-button');
+projectListButton.append(lastButton);
+
+const projectButton1 = document.createElement('button');
+projectButton1.setAttribute('type', 'button');
+projectButton1.setAttribute('class', 'btn');
+projectButton1.innerText = 'See live';
+const seeLive = document.createElement('img');
+seeLive.setAttribute('alt', 'see live');
+seeLive.setAttribute('class', 'btn-1-img');
+seeLive.src = './public/arrow-up.png';
+projectButton1.append(seeLive);
+lastButton.append(projectButton1);
+
+const projectButton2 = document.createElement('button');
+projectButton2.setAttribute('type', 'button');
+projectButton2.setAttribute('class', 'btn');
+projectButton2.innerText = 'See source';
+const seeSource = document.createElement('img');
+seeSource.setAttribute('alt', 'see source');
+seeSource.setAttribute('class', 'btn-2-img');
+seeSource.src = './public/SVG-1.svg';
+projectButton2.append(seeSource);
+lastButton.append(projectButton2);
+
+function hidePopup() {
+  const popup = document.getElementById('popup');
+  popup.style.display = 'none';
+}
+closeButton.addEventListener('click', hidePopup);
+
+// function checkMediaQuery() {
+//     if (window.matchMedia("(min-width: 768px)").matches) {
+//         console.log("Media query matches (screen width is >= 768px)");
+//         projectImg1.style.display = "block";
+//         projectImg2.style.display = "none";
+//         }
+//     else {
+//             projectImg1.style.display = "none";
+//             projectImg2.style.display = "block";
+//     }
+// }
+// checkMediaQuery();
+// popup.addEventListener('resize', checkMediaQuery);
```

Comment:
- [ ]  Kindly consider removing all the old pieces of code commented. they make your project looks messy. If you need to review a previous version of your code, you can always use git history.